README.md

```diff
@@ -35,23 +35,25 @@ Install and run `nox` to run all tests. That's it for simple contributions!
 `nox` will create virtual environments in `.nox` inside the checked out project
 and install the requirements needed to run the tests there.
 
-
 ---
 
-antsibull-docs depends on the sister antsibull-core and antsibull-docs-parser projects.
-By default, `nox` will install a development version of these projects from
-Github.
-If you're hacking on antsibull-core and/or antsibull-docs-parser alongside antsibull-docs,
-nox will automatically install the projects from `../antsibull-core` and
-`../antsibull-docs-parser` when running tests if those paths exist.
+antsibull-docs depends on the sister antsibull-changelog, antsibull-core, and
+antsibull-docs-parser projects. By default, `nox` will install a development version of
+these projects from Github.
```

Comment:
I wrote this section originally to adhere to sembr.org.

```suggestion
antsibull-docs depends on the sister antsibull-changelog, antsibull-core, and
antsibull-docs-parser projects.
By default, `nox` will install a development version of these projects from Github.
```

Reply:
ecc0c55f74c0a7cff7d961b53ca1e11b2e51b32f.